projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -3737,14 +3670,33 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
     }
 
     public get outerWidth() {
-        return this.hasVerticalSroll() ? this.calcWidth + 18 : this.calcWidth;
+        return this.hasVerticalSroll() ? this.calcWidth + 1 + this.scrollWidth : this.calcWidth + 1;
+    }
+
+    /**
+     * @hidden
+     * Gets the visible content height that includes header + tbody + footer.
+     */
+    public getVisibleContentHeight() {
+        let height = this.theadRow.nativeElement.clientHeight + this.tbody.nativeElement.clientHeight;
+        if (this.hasSummarizedColumns) {
```

Comment:
The check can be omitted yes. 

Yes since the toolbar suggests that it has tools for managing the grid user content and since the method returns the parts of the grid that usually has user defined information and that's why the method got called `getVisibleContentHeight`. It is currently mainly used in resizing to size the drag indicator from the top of headers to footer.

Reply:
@skrustev OK, make it a getter.

Reply:
@kdinev Currently when using getter child class cannot access the superclass getter in order to get its value. That's why it is method so it can be overridden and be able to call super.getVisibleContentHeight().